### PR TITLE
Portfolio blurb written along with some styling

### PR DESCRIPTION
**Pull request by cel243**
I added a paragraph talking about myself along with two links. I also uploaded several photos of my dog, but right now I am only using one to display on the webpage. I did some styling on the text and also added a background image for the page.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<div id="white-box">
+			<h1>Caroline's Portfolio</h1>
+			<div id="blurb">
+				<p>I'm Caroline and this is my portfolio! A bit about me: I'm currtenly a rising Junior at Cornell's
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/index.html:16]: nit: typo :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+					College
+					of
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/index.html:18]: this section is hard to follow in code due to code formatting. This YAQS suggests https://yaqs.corp.google.com/eng/q/5051660848332800 a line limit of 100, could you try to keep it to that?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,51 @@
...
+					engineering, and I can't wait to see what more I'll learn during this internship! </p>
+			</div>
+		</div>
+		<p> Come check me out on <a href="https://www.linkedin.com/in/caroline-lui-877638174/"
+				target="_blank">LinkedIn</a> and <a href="https://github.com/cel243" target="_blank">GitHub</a>!
+		</p>
+		<p>Below is my dog, Penny!</p>
+		<div id=penny-image>
+			<a href="images/Penny-1.JPG"><img src="images/Penny-1.JPG"/></a>
+
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/index.html:46]: nit: could you please get rid of this extra new line?

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html]: fixed!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html]: format has been fixed!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html]: new line is gone now!

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -11,41 +11,40 @@
...
 		</p>
 		<p>Below is my dog, Penny!</p>
 		<div id=penny-image>
 			<a href="images/Penny-1.JPG"><img src="images/Penny-1.JPG"/></a>
-
 		</div>
-	</div>
 </body>
 
 </html>
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/index.html]: could you please add a newline at the end of this file? It's a common practice so that future additions beyond this will not show as an additional diff line in the future.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -11,41 +11,40 @@
...
 			</div>
 		</div>
-		<p> Come check me out on <a href="https://www.linkedin.com/in/caroline-lui-877638174/"
-				target="_blank">LinkedIn</a> and <a href="https://github.com/cel243" target="_blank">GitHub</a>!
+		<p> Come check me out on
+			<a href="https://www.linkedin.com/in/caroline-lui-877638174/" target="_blank">LinkedIn</a> and
+			<a href="https://github.com/cel243" target="_blank">GitHub</a>!
 		</p>
 		<p>Below is my dog, Penny!</p>
 		<div id=penny-image>
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/index.html:46]: I believe that id=penny-image needs some quotes

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,9 +1,30 @@
...
+
+#blurb {
+    text-align: justify; 
+}
+
+
+#penny-image img {
+    max-height: 500px;
+    max-width: 500px;
+}
```
Comment (davidgaskins) [inline portfolio/src/main/webapp/style.css:30]: same here for the newline

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html]: sure thing! I'll add a new line at the bottom of every file

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html:46]: will add that and recommit ASAP

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -1,18 +1,51 @@
...
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+
+<head>
```
Comment (yutaow) [inline portfolio/src/main/webapp/index.html:4]: Could you check the indent from this line?
Ideally the un-changed line should remain white in this diff view instead of showing the same lines deleted and added back.

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -1,9 +1,30 @@
+body {
+    background-image: url("images/paper-background.jpg");
+    background-repeat: no-repeat;
+    background-size: cover; 
+}
+
+
```
Comment (yutaow) [inline portfolio/src/main/webapp/style.css:7]: Why extra line?

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -1,9 +1,30 @@
...
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    text-align: center;
```
Comment (yutaow) [inline portfolio/src/main/webapp/style.css:9]: Again, the indent seems off in this file

File: portfolio/src/main/webapp/style.css [hunk 10]
```diff
@@ -1,9 +1,30 @@
...
+    padding: 1em; 
+}
+
+#blurb {
+    text-align: justify; 
 }
 
-#greeting-container {
-  margin-top: 20px;
+
```
Comment (yutaow) [inline portfolio/src/main/webapp/style.css:26]: Ditto, one extra line.

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (cel243) [reply portfolio/src/main/webapp/index.html:4]: my cloud shell auto-formats to this indentation, so hopefully it will be consistent from here on out!

File: portfolio/src/main/webapp/style.css (hunk 8 above)
Comment (cel243) [reply portfolio/src/main/webapp/style.css:7]: will remove ASAP!

File: portfolio/src/main/webapp/style.css (hunk 10 above)
Comment (cel243) [reply portfolio/src/main/webapp/style.css:26]: will remove ASAP!
